packages/ts-codegen/README.md

```diff
@@ -1,3 +1,10 @@
-# @graphitation/graphql-codegen
+# @graphitation/ts-codegen
+
+Package is used for [supermassive](../cli/src/supermassive.ts) cli to generate typescript code from a GraphQL document.
+
+### Parameters
+
+- `document` (DocumentNode): The GraphQL document to generate TypeScript code from.
+- `options` (object): An object containing various options for the code generation.
```

Comment:
Could you add a codeblock with option type defs here?

Reply:
I just copy past the interface. I guess this is anyway used via CLI.